test/lib/api.spec.js

```diff
@@ -195,4 +195,30 @@ describe('api', () => {
       assert.equal(cacheGetSpy.callCount, 0);      // cache not used
     });
   });
+
+  describe('available', async () => {
+
+    beforeEach(() => sinon.stub(environment, 'apiUrl').get(() => 'http://api/medic'));
+
+    it('should return true in no error found in request', async () => {
+      sinon.stub(environment, 'isArchiveMode').get(() => false);
+      sinon.stub(mockRequest, 'get').resolves('okey dokey');
+      sinon.stub(log, 'error');
+      let isAvailable = await api().available();
```

Comment:
Please always use `const` over `let`. 
```suggestion
      const isAvailable = await api().available();
```